File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        rptr, wptr = 0, 0
        while rptr < len(nums):
            if rptr > 0 and nums[rptr] == nums[rptr-1]:
                rptr += 1
            else:
                nums[wptr] = nums[rptr]
                wptr += 1
                rptr += 1
        return wptr

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let rptr = 0, wptr = 0
    while (rptr < nums.length ) {
        if (rptr > 0 && nums[rptr] === nums[rptr-1]) rptr++
        else nums[wptr++] = nums[rptr++]
    }
    return wptr
};